lib/utils/oclif-utils.ts

```diff
@@ -17,11 +17,6 @@
 
 import * as Config from '@oclif/config';
 
-export const convertedCommands = {
-	'env:add': 'env add',
-	'env:rm': 'env rm',
```

Comment:
This object was not used / not referenced in the code, hence deleted (it had been left behind from a previous refactoring).